crates/api_crud/src/user/delete.rs

```diff
@@ -19,9 +19,13 @@ use lemmy_utils::error::{LemmyErrorType, LemmyResult};
 pub async fn delete_account(
   data: Json<DeleteAccount>,
   context: Data<LemmyContext>,
-  local_user_view: LocalUserView,
```

Comment:
@Nothing4You noted that since logins are invalidated after a site ban, we can't use the local user view here.

Reply:
i didn't realize you could just turn that into an `Option` and it just works, that's neat.
is that standard actix web behavior or implemented elsewhere?

Reply:
Its used for the modlog, random, and a lot of the open reads for things like posts, communities. 

As far as I can tell from [here](https://github.com/LemmyNet/lemmy/blob/main/crates/routes/src/middleware/session.rs#L76), since we're only inserting the `LocalUserView` and not dealing with options, then actix can by default handle options in its request handlers.